corehq/util/pagination.py

```diff
@@ -195,7 +196,7 @@ def __iter__(self):
 
         retried = {}
         while self.state.retry != retried:
-            for item_id, retries in list(self.state.retry.iteritems()):
+            for item_id, retries in list(six.iteritems(self.state.retry)):
```

Comment:
why does this need to be wrapped in `list`

Reply:
```list``` looks unnecessary to me.

@snopoke was there a reason ```list``` was used here?

Reply:
Can't think why I'd have added the `list`. Doesn't seem necessary.